doc/tools/coqrst/coqdoc/main.py

```diff
@@ -35,7 +36,15 @@
 
 def coqdoc(coq_code, coqdoc_bin=None):
```

Comment:
It's kinda weird that using an explicit argument expects `coqdoc` but without it it will use `rocq`
shall we change to expecting explicit `rocq_bin`?